app/dashboard/views/custom_alias.py

```diff
@@ -250,6 +252,20 @@ def custom_alias():
     )
 
 
+def get_suffix(user: User) -> str:
```

Comment:
Hey I wonder if we should replace `random_word()` by this method in other places, for example https://github.com/simple-login/app/blob/30183ac8c3f705cac7d834dfe811b96b56ff81bc/app/models.py#L1127 ?

Reply:
I'm not sure if that will have any side effect because it looks like the implementation is a little different elsewhere (for example, there doesn't seem to be a signature of suffix?)

Reply:
The `random_word` function is mostly used for generating an alias suffix so I guess we can almost replace it everywhere by `get_suffix()`